### DISCUSSION TITLE
How to set `reportMissingParameterType = true` in `pyproject.toml` for all directories except for `tests/`?

### DISCUSSION THREAD

**Discussion by rafrafek** [microsoft/pyright#7131] [Q&A]
Hello,
How to set `reportMissingParameterType = true` in `pyproject.toml` for all project directories except for my `tests/` directory?

**Comment by erictraut** [comment]
You have a few options:
1. Use file-level pyright comments to suppress the error in specific files (`# pyright: reportMissingParameterType=false` at the top of the file). This is the technique I use in my code.
2. Create separate pyrightconfig.json files for different parts of your project. Use multi-root workspaces to set up a separate workspace, each with different configurations. I find this to be somewhat more cumbersome. Plus, some editors don't have great support for multi-root workspaces.
